tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Builders/OrganisationBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NHSD.BuyingCatalogue.Identity.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.UnitTests.Builders
+{
+    internal sealed class OrganisationBuilder
+    {
+        internal static OrganisationBuilder Create() => new OrganisationBuilder();
+
+        private Guid _organisationId;
+        private string _name;
+
+        public OrganisationBuilder()
+        {
+            _organisationId = new Guid();
```

Comment:
Did you want to create a new guid here? If so use `Guid.New()`, otherwise I think it defaults to an empty one.